src/main/java/de/tum/in/www1/artemis/web/rest/ExamResource.java

```diff
@@ -239,6 +240,67 @@ public ResponseEntity<Exam> updateExam(@PathVariable Long courseId, @RequestBody
         return ResponseEntity.ok(savedExam);
     }
 
+    /**
+     * PATCH /courses/{courseId}/exams/{examId}/student-exams/working-time : Update the working time of all exams
+     *
+     * @param courseId          the course ID to which the exam belong to
+     * @param examId            the exam ID the working time should be extended for
+     * @param workingTimeChange the working time change in seconds (can be positive or negative, but must not be 0)
+     * @return the ResponseEntity with status 200 (OK) and with the updated exam as body
+     */
+    @PatchMapping("/courses/{courseId}/exams/{examId}/student-exams/working-time")
+    @EnforceAtLeastInstructor
+    public ResponseEntity<Exam> updateExamWorkingTime(@PathVariable Long courseId, @PathVariable Long examId, @RequestBody Integer workingTimeChange) {
+        log.debug("REST request to update the working time of exam : {}", examId);
+
+        examAccessService.checkCourseAndExamAccessForInstructorElseThrow(courseId, examId);
+
+        if (workingTimeChange == 0) {
+            throw new BadRequestException();
+        }
+
+        Exam exam = examService.findByIdWithExerciseGroupsAndExercisesElseThrow(examId);
```

Comment:
It seems like the exercises are not actually needed, so we should not load them here.

Reply:
Replaced by `examRepository.findByIdElseThrow`